src/JwksClient.js

```diff
@@ -66,7 +66,7 @@ export class JwksClient {
       }
 
       const signingKeys = keys
-        .filter(key => key.use === 'sig' && key.kty === 'RSA' && key.kid && ((key.x5c && key.x5c.length) || (key.n && key.e)))
+        .filter(key => (!key.use || key.use === 'sig') && key.kty === 'RSA' && key.kid && ((key.x5c && key.x5c.length) || (key.n && key.e)))
```

Comment:
Is there a way we can improve this filtering condition? Each clause we add makes it longer and harder to read.

Reply:
Please refer comment on other thread, https://github.com/auth0/node-jwks-rsa/pull/116#discussion_r365780322

Once we finalize that, I will try to improve the checks to make it more readable

Reply:
One possible way to simplify filter function is following, i.e. break && conditions into multiple if statements

```
const signingKeys = keys.filter((key) => {
    if (key.kty !== 'RSA') {
        return false;
    }
    if (!key.kid) {
        return false;
    }
    if (key.hasOwnProperty('use') && key.use !== 'sig') {
        return false;
    }
    return ((key.x5c && key.x5c.length) || (key.n && key.e));
})
```
Your thoughts on this?

Reply:
Yes, it does. Go ahead 👍

Reply:
Done.